cubeseed_login/src/component/tables/NewFarmersTable.tsx

```diff
@@ -0,0 +1,312 @@
+    },
+    {
+      id: 4,
+      name: "farmer4",
+      email: "farmer4@gmail.com",
+      time: "5/1/23",
+      certificate: "document.pdf",
+      code: null,
+      document1: "Utility Bill",
+      document2: "Picture of Farm",
+    },
+    {
+      id: 5,
+      name: "farmer5",
+      email: "farmer5@gmail.com",
+      time: "5/1/23",
+      certificate: null,
+      code: 1234,
+      document1: "Picture of Farm",
+      document2: "Document of Land Ownership",
+    },
+    {
+      id: 6,
+      name: "farmer6",
+      email: "farmer6@gmail.com",
+      time: "5/1/23",
+      certificate: null,
+      code: 1234,
+      document1: "Document of Land Ownership",
+      document2: "Utility Bill",
+    },
+    {
+      id: 7,
+      name: "farmer7",
+      email: "farmer7@gmail.com",
+      time: "5/1/23",
+      certificate: "document.pdf",
+      code: null,
+      document1: "Utility Bill",
+      document2: "Picture of Farm",
+    },
+    {
+      id: 8,
+      name: "farmer8",
+      email: "farmer8@gmail.com",
+      time: "5/1/23",
+      certificate: null,
+      code: 1234,
+      document1: "Picture of Farm",
+      document2: "Document of Land Ownership",
+    },
+    {
+      id: 9,
+      name: "farmer9",
+      email: "farmer9@gmail.com",
+      time: "5/1/23",
+      certificate: "document.pdf",
+      code: null,
+      document1: "Document of Land Ownership",
+      document2: "Utility Bill",
+    },
+    {
+      id: 10,
+      name: "farmer10",
+      email: "farmer10@gmail.com",
+      time: "5/1/23",
+      certificate: "document.pdf",
+      code: null,
+      document1: "Utility Bill",
+      document2: "Picture of Farm",
+    },
+    {
+      id: 11,
+      name: "farmer11",
+      email: "farmer11@gmail.com",
+      time: "5/1/23",
+      certificate: "document.pdf",
+      code: null,
+      document1: "Picture of Farm",
+      document2: "Document of Land Ownership",
+    },
+    {
+      id: 12,
+      name: "farmer12",
+      email: "farmer12@gmail.com",
+      time: "5/1/23",
+      certificate: null,
+      code: 1234,
+      document1: "Document of Land Ownership",
+      document2: "Utility Bill",
+    },
+    {
+      id: 13,
+      name: "farmer13",
+      email: "farmer13@gmail.com",
+      time: "5/1/23",
+      certificate: null,
+      code: 1234,
+      document1: "Utility Bill",
+      document2: "Picture of Farm",
+    },
+    {
+      id: 14,
+      name: "farmer14",
+      email: "farmer14@gmail.com",
+      time: "5/1/23",
+      certificate: "document.pdf",
+      code: null,
+      document1: "Picture of Farm",
+      document2: "Document of Land Ownership",
+    },
+    {
+      id: 15,
+      name: "farmer15",
+      email: "farmer15@gmail.com",
+      time: "5/1/23",
+      certificate: null,
+      code: 1234,
+      document1: "Document of Land Ownership",
+      document2: "Utility Bill",
+    },
+  ]
+
+  const [revealButtons, setRevealButtons] = useState(false)
```

Comment:
and maybe a type check for the revealButton state as well

`const [revealButtons, setRevealButtons] = useState<boolean>(false)`